frontend/src/scenes/data-warehouse/dataWarehouseOverviewLogic.ts

```diff
@@ -0,0 +1,114 @@
+import { actions, afterMount, kea, listeners, path, reducers, selectors } from 'kea'
```

Comment:
I've added a couple of comments here but ultimately this logic should be empty until we need some functionality. There's no point in having a "boilerplate" logic with code that isn't used - unused code is bad code. 

This new scene/page is gonna be quite complex in terms of the logic/data needs and so we should be planning out the kea logic architecture of it before it ends up being an unwieldy mess like so many other kea logics we have (see `multitabEditorLogic`, `dataNodeLogic`, `insightSceneLogic`, `insightDataLogic`, etc etc). 

What this actually means in practice is figuring out ahead of time the sort of data models we will be wanting to load on this page, making any backend adjustments to make data pulling simpler, and compartmentalizing the logics to support this